packages/shaku-code-annotate-shiki/src/codeToShakuHtml.ts

```diff
@@ -357,6 +360,46 @@ export let codeToShakuHtml = function (
   };
 };
 
+const commentPatternMap: { [key: string]: string[] } = {
+  js: ["/*", "*/", "//"],
+  html: ["<!--", "-->", ""],
```

Comment:
https://github.com/JSerZANP/shaku/blob/main/packages/shaku-code-annotate-shiki/src/defaultCode.ts

How about 150+ more languages? 🤔

Reply:
We can add more patterns here if more lang.
How do you think or any other idea to solve this kind of issue?

Reply:
Are we able to detect following pattern?

1. consecutive lines of comments
2. all lines are shaku lines, except the last line is an normal comment with empty body.

if this is possible, then we might not need to hardcode patterns by languages.
this might be helpful https://github.com/JSerZANP/shaku/blob/main/packages/shaku-code-annotate-shiki/src/codeToShakuHtml.ts#L382-L479